ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -148,6 +148,14 @@ adParameter
     | (ANOMALY_SCORE_THRESHOLD EQUAL anomaly_score_threshold=decimalLiteral)
     ;
 
+mlCommand
```

Comment:
does ml also no arguments? e.g. source = index | ml. The syntax actaully allow this.

Reply:
If no action or algorithm arguments, it will throw exception. I don't think we have to restrict the command content at command parsing stage, we leave all parameters validation to ml-client.